data/helpers.d/network

```diff
@@ -0,0 +1,18 @@
+# Normalize the path syntax
+# Check the slash at the beginning of path and its absence at ending
+#
+# example: path=$(ynh_normalize_path $path)
+#
+# usage: ynh_normalize_path path_to_normalize
+# | arg: path_to_normalize - Path to normalize before using it
+ynh_normalize_path () {
+	path=$1
+	test -n "$path" || ynh_die "The path variable is empty."
```

Comment:
The error message doesn't make sens, it should be something like "Error: ynh_normalize_pathis expecting a path as argument"